Automattic/wp-calypso
# My Plan: Improve Plan Icon Display

#### Changes proposed in this Pull Request

This improves the My Plan display at `/plans/my-plan/site` for mobile. I think the screenshots explain better than the words for this.

#### Testing instructions

Try testing a variety of cases, including with the Jetpack Free plan (to ensure #33883 doesn't return, especially since this changes @scottsweb's fix):

**Before:**
<img width="505" alt="Screenshot 2019-08-17 at 12 13 42" src="https://user-images.githubusercontent.com/43215253/63210978-3fb12e00-c0e9-11e9-87c8-613e9747db15.png">

**After:**
<img width="503" alt="Screenshot 2019-08-17 at 12 13 47" src="https://user-images.githubusercontent.com/43215253/63210982-450e7880-c0e9-11e9-978d-e038b7fe283d.png">

**Before:**
<img width="512" alt="Screenshot 2019-08-17 at 12 14 17" src="https://user-images.githubusercontent.com/43215253/63210986-4dff4a00-c0e9-11e9-92e6-537af78b4e7b.png">

**After:**
<img width="509" alt="Screenshot 2019-08-17 at 12 14 14" src="https://user-images.githubusercontent.com/43215253/63210991-52c3fe00-c0e9-11e9-9919-dd555fa5abf3.png">

The plan icon should always be centred. 

<img width="613" alt="Screenshot 2019-08-17 at 11 59 37" src="https://user-images.githubusercontent.com/43215253/63211001-6bccaf00-c0e9-11e9-82c6-841e06bd3268.png">
<img width="620" alt="Screenshot 2019-08-17 at 11 58 59" src="https://user-images.githubusercontent.com/43215253/63211009-77b87100-c0e9-11e9-9e75-8902e50d533e.png">

Fixes #35507
